src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.command;
+
+import duke.TaskList;
+
+/**
+ * Represents command to mark a specific task complete.
+ */
+public class DoneCommand extends Command {
+
+    public DoneCommand() {
+        this.commandText = "done";
+    }
+    /**
+     * Marks the particular task complete.
```

Comment:
Should there be an empty like between the method description and parameters section?